src/main/pages/setup-docker/docker-cli.adoc

```diff
@@ -8,142 +8,216 @@ folder: setup-docker
+The `backup` command creates `TAR` files from the `/instance` directory and places them in the `/backup` directory. These files are restoration-ready.
 
-* `init` : Initialize the directory specified and add a default `Chefile` file if there is none
-* `up` : Boot Eclipse Che with workspace on folder
-* `down` : Stop Eclipse Che and any workspaces
-* `ssh` : Connect to the running workspace by using ssh
-* `status` : Display if an instance of Eclipse Che is running or not for the specified folder.
+*_config_*
 
-'''''
+The `config` command generates a Che instance configuration that is placed in the `/instance` directory. It uses Puppet to generate Docker Compose configuration files to run Che and its associated server. Che’s server configuration is generated as a `che.properties` file that is mounted in the Che server when it boots. This command is executed on every `start` or `restart`.
 
-_download_
+If you are using a `eclipse/che:<version>` image and it does not match the version in the `/instance/che.ver` file, the configuration aborts to prevent you from running a configuration for a different version.
 
-Used to download Docker images that will be stored in your Docker images repository. This command downloads images that are used by the CLI as utilities, for Che to do initialization and configuration, and for the runtime images that Che needs when it starts. This command respects `--offline`, `--pull`, `--force`, and `--no-force` (default). This command is invoked by `che init`, `che config`, and `che start`.
+It respects `--no-force`, `--pull`, `--force`, and `--offline` command options.
 
-`download` is invoked by `che init` before initialization to download images for the version specified by `eclipse/che:<version>`.
+*_destroy_*
 
-It is possible to override the docker images used by the CLI by setting the following environment variables:
+The `destroy` command deletes the `/docs`, `che.env` and `/instance` directories, including user workspaces, projects, data, and user database. To skip the confirmation warning message, pass the `--quiet` argument in the command. To delete the `cli.log` file, pass the `--cli` argument. By default, the `cli-log` file is retained for traceability.
 
-* `IMAGE_INIT` to override the default `eclipse/che-init:<version>` docker image,
+*_dir_*
+
+The `dir` command boots a new Eclipse Che instance with a workspace for the `:/chedir` directory defined as volume mount in the parameter.
+
+For example, if you give `$HOME/my-project` as a parameter, a new Che instance is created using `$HOME/my-project` as a project in the IDE. Inside the IDE, the `/projects` directory contains a `my-project` directory with your host directory. Any changes inside the IDE are reflected in your host directory. And, updating a file on your local computer updates the content of the file inside the IDE.
+
+* `init`: Initializes the directory specified and adds a default `Chefile` file if there is none.
```

Comment:
While this is technically correct, "`Chefile` file" sounds a bit silly. ;)

Reply:
Agree :)